### ISSUE TITLE
Publisher Name Error on Microsoft Store verification

### ISSUE THREAD

**Issue by jaimevaqueiro** [YehudaKremer/msix#42]
I'm trying to upload the .msix package to the Windows store, but verification fails with the following error messages:

- Invalid package family name:  PadmitS.A.P.I.deC.V.WardHome_pjyac4g0caf2w (expecting: PadmitS.A.P.I.deC.V.WardHome_1hxjgg9m0hxqm)
- Invalid package publisher name:  CN=Msix Testing, O=Msix Testing Corporation, C=US (expecting: CN=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX)
	
What's strange is that I've defined the publisher with the correct value in pubspec.yaml:

```
msix_config:
  display_name: Ward Home
  publisher_display_name: Padmit, S.A.P.I. de C.V.
  identity_name: PadmitS.A.P.I.deC.V.WardHome
  msix_version: 1.0.0.0
  publisher: CN=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
  logo_path: C:\Users\Padmit\Documents\Ward\admin\home_logo.png
  start_menu_icon_path: C:\Users\Padmit\Documents\Ward\admin\home_logo.png
  tile_icon_path: C:\Users\Padmit\Documents\Ward\admin\home_logo.png
  icons_background_color: transparent
  languages: es
  architecture: x64
  capabilities: 'internetClient,privateNetworkClientServer'
```
Any ideas as to what I might be doing wrong?

I'm building the app with the following commands:

```
flutter build windows
flutter pub run msix:create
```
Thanks

**Comment by jaimevaqueiro** [comment]
I guess the question is how do I make sure `flutter pub run msix:create` uses the right publisher information? I've copied mine directly from Microsoft Store, same thing for the `identity_name `

**Comment by YehudaKremer** [comment]
Hello @jaimevaqueiro 
This bug prevents advertising to the store, I will fix it as soon as possible
Thank you for this report

**Comment by YehudaKremer** [comment]
Hey

i published new version `2.1.0` with a new flag `--store` for generation msix installer for Windows Store,
and updated the documentation:
![image](https://user-images.githubusercontent.com/946652/117422374-32c11780-af28-11eb-9536-87d2e1ea9d5a.png)

@jaimevaqueiro please update if its help your publish situation
